Controllers/sessions.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+const User = require('../Models/userModel')
+
+router.get('/sessions', (req, res) => {
+  console.log(`session get router`)
+})
+
+// USER LOGIN ROUTE (CREATE SESSION ROUTE)
+router.post("/sessions", (req, res) => {
```

Comment:
Wasn't there already a log in route?

Reply:
I deleted sessions.js in the last commit